javascript/ql/src/semmle/javascript/frameworks/SQL.qll

```diff
@@ -87,8 +80,9 @@ private module MySql {
     string kind;
 
     Credentials() {
-      exists(string prop |
-        this = [createConnection(), createPool()].getOptionArgument(0, prop).asExpr() and
+      exists(API::Feature call, string prop |
+        (call = createConnection() or call = createPool()) and
```

Comment:
Is there some reason to go from set literal to `(... or ...)` here?

Reply:
No; set literal would be much better.